LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Koa contributors
```

Comment:
```suggestion
Copyright (c) 2014-present Koajs contributors
```

Reply:
Will update this when I’m back online tomorrow